user-guide/Advanced_Modules/DOM/tutorials/DOM_Making_DOM_Stateful.md

```diff
@@ -0,0 +1,287 @@
+---
+uid: DOM_Making_DOM_Stateful
+---
+
+# Making DOM objects state-aware
+
+This tutorial shows how you can make your DOM objects state-aware. This allows a DOM instance to behave differently depending on the state it transitions to. The following things can be defined for each state:
+
+- Which fields are **visible** to the user.
+- Which fields are **read-only** for the user.
+- Which fields are **mandatory**.
+- To which state the instance can **transition**.
+- What **actions** are allowed.
+- What **buttons** to show on the form.
+
+The content and screenshots for this tutorial were created in DataMiner version 10.4.3.
+
+Expected duration: 20 minutes
+
+<!-- Uncomment this section when the link to the Kata is available:
```

Comment:
As the link to the Kata is not yet available, I've commented out this block for now. As soon as the link is available, we can uncomment it.